cfme/configure/settings.py

```diff
@@ -327,25 +391,66 @@ def __init__(self, name=None, filters=None, appliance=None):
         self.name = name
         self.filters = filters or []
 
-    def update(self, updates, expect_success=True):
+    def update(self, updates):
         navigate_to(self, 'All')
-        fill(self.filter_form, {'filter_tree': updates.get('filters')},
-             action=form_buttons.save)
-        if expect_success:
-            flash.assert_success_message('Default Filters saved successfully')
+        view = navigate_to(self, 'Edit')
+        for value in updates['filters']:
+            for path in value:
+                if type(path) == list and view.tree.fill(path):
+                    view.save.click()
 
 
 @navigator.register(DefaultFilter, 'All')
 class DefaultFilterAll(CFMENavigateStep):
+    VIEW = MySettingsView
```

Comment:
**REQUIRED**
Why do you return `MySettingsView` if this step supposed to return all default filters?

Reply:
I am using that MySettingView to navigate to the required tab.

Reply:
Will make changes.